SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditor.cpp

```diff
@@ -212,8 +216,30 @@ void FSpatialGDKEditor::GenerateSnapshot(UWorld* World, FString SnapshotFilename
 	}
 }
 
+void FSpatialGDKEditor::WarnIfManualWorkerConnectionSet(FString LaunchConfigPath)
+{
+	FString FileContents;
+	FFileHelper::LoadFileToString(FileContents, LaunchConfigPath.GetCharArray().GetData());
+
+	const FRegexPattern ManualWorkerFlagPattern("\"manual_worker_connection_only\" *: *true");
+	FRegexMatcher ManualWorkerFlagMatcher(ManualWorkerFlagPattern, FileContents);
+
+	if (ManualWorkerFlagMatcher.FindNext())
+	{
+		UE_LOG(LogSpatialGDKEditor, Warning, TEXT("A cloud deployment with \"manual_worker_connection_only\" set to true was launched. This means the server worker will need to be connected manually."));
+		
+		FNotificationInfo Info(FText::FromString(TEXT("Manual worker connection only set for cloud build")));
```

Comment:
I wonder if this is enough. Most designers don't have the output log open and so will miss the message on how to fix it. 
I suggest we add a confirmation window the same as what we did with the blocking port. 
Notify the user and if they agree then carry on, otherwise abandon the launch.